scripts/sanitycheck

```diff
@@ -662,7 +662,7 @@ class SizeCalculator:
                    "kobject_data", "mmu_tables", "app_pad", "priv_stacks",
                    "ccm_data", "usb_descriptor", "usb_data", "usb_bos_desc",
                    'log_backends_sections', 'log_dynamic_sections',
-                   'log_const_sections']
+                   'log_const_sections',"app_smem"]
```

Comment:
I noticed that this is now using two different quoting styles (" vs '), not sure why (you didn't introduce it and were using the original style but subsequent commits have switched to single quotes) or what's preferred.